seed/templates/seed/_header.html

```diff
@@ -59,11 +59,11 @@
             >Data Quality</a
           >
         </li>
-        <li><a ui-sref="organization_derived_columns({organization_id: menu.user.organization.id, inventory_type: 'properties'})" ui-sref-active="active" translate>Derived Columns</a></li>
-        <li><a ui-sref="organization_email_templates({organization_id: menu.user.organization.id})" ui-sref-active="active" translate>Email Templates</a></li>
+        <li><a ui-sref="organization_derived_columns({organization_id: menu.user.organization.id, inventory_type: 'properties'})" ng-if="menu.user.organization.is_parent && auth.requires_owner" ui-sref-active="active" translate>Derived Columns</a></li>
```

Comment:
This is technically redundant as owners only belong in the parent. 

That okay tho, as this is more explicit and future proof. The bug will persist on develop, tho